volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -92,53 +92,94 @@ def get_sections(self):
         else:
             num_sects = self._get_sect_count(self.sect_attrs.grp)
         arr = self._context.object(
-            self.get_symbol_table().name + constants.BANG + "array",
+            self.get_symbol_table_name() + constants.BANG + "array",
             layer_name=self.vol.layer_name,
             offset=self.sect_attrs.attrs.vol.offset,
             subtype=self._context.symbol_space.get_type(
-                self.get_symbol_table().name + constants.BANG + "module_sect_attr"
+                self.get_symbol_table_name() + constants.BANG + "module_sect_attr"
             ),
             count=num_sects,
         )
 
         for attr in arr:
             yield attr
 
-    def get_symbols(self):
```

Comment:
Removing existing functions will require a major version bump as well.  Since you've defined new functions, please just include a deprecation warning and keep these methods around so the API doesn't change.

Reply:
Ah sorry, the diff was confusing...